### ISSUE TITLE
Add ability to control the amount of spacing around a Header.

### ISSUE THREAD

**Issue by Taliesin1** [SlRvb/Obsidian--ITS-Theme#254]
### Is your feature request related to a problem? Please describe.
Headers should always be closer to the paragraph that follows it than the paragraph that precedes it. This is due to the gestalt principle of "proximity" in design. Although this has been impossible on the web until relatively recently, it's a snap to do now. 

### Describe the solution you'd like
It would be great if you could add a parameter for users to be able to set the amount of spacing after a header. Really before _and_ after would be ideal, but I'd be happy with after only.

### Describe alternatives you've considered
No alternatives. The design and usability of the content will not be optimized if headers float halfway between the paragraphs.

Thanks so much for your consideration!

**Comment by SlRvb** [comment]
I've added this to ITS v1.1.38 :) Sliders should be under `Note > Headers > Advanced Heading Options`

**Comment by Taliesin1** [comment]
Wow, that was quick! Thanks!

Playing with it now, I can't get the Heading Top Spacing slider to do anything with the styles. The Heading Bottom Spacing slider works fine.

I don't know if the Heading Top Spacing is somehow interactive with the Header Spacing control, but experimenting with any combination of these two resulted in no more change than if I just used Heading Spacing alone.

That said, this did solve the issue, even as-is. Between tweaking the Header Spacing and the Heading Bottom Spacing, I was able to get a pretty good result, much better than when the heading floats in the middle between two paragaphs. 

But I thought I'd alert you that the Heading Top Spacing seems to not do anything. Probably use error on my part, but there it is. 

⭐️⭐️⭐️⭐️⭐️
